lib/api/repair/v1/service.test.ts

```diff
@@ -0,0 +1,87 @@
+import { config } from "@mtfh/common/lib/config";
+import { useAxiosSWR } from "@mtfh/common/lib/http";
+import { useRepairs } from "./service";
+
+import { RepairsFilters } from "./types";
+
+jest.mock("@mtfh/common/lib/http", () => ({
+  ...jest.requireActual("@mtfh/common/lib/http"),
+  useAxiosSWR: jest.fn(),
+}));
+
+const repairsMock = [
```

Comment:
Ideally we want to generate mocks from `mtfh-frontend` which I also need for my PR. Let's catch up on that.

Reply:
Updated and using generated mocks instead